edx_proctoring/backends/rest.py

```diff
@@ -76,7 +77,11 @@ def get_javascript(self):
         Returns the url of the javascript bundle into which the provider's JS will be loaded
         """
         package = self.__class__.__module__.split('.')[0]
-        bundle_chunks = get_files(package, config="WORKERS")
+        bundle_chunks = ''
+        try:
+            bundle_chunks = get_files(package, config="WORKERS")
+        except BaseWebpackLoaderException:
+            pass
```

Comment:
It's generally not great to swallow exceptions with `pass`. You can emit a warning, instead, so that somebody looking at the logs can understand what might be going on.

And some comments explaining what's going on would be good.

Also, you can do `try: ... except: ... else:` here

Reply:
I don't actually see a benefit of try-except-else, but I appreciate you letting me know that that's a thing.